index.html

```diff
@@ -1,9 +1,90 @@
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>NPM Clone</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+      <header>
+        <div>
+            <div>
+              <img class="blackHeart" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\black-heart.png" alt="black heart" />
+              <span>Nifty Penguin Magic</span>
+            </div>
+            <nav id="nav-bar">
+              <ul>
+                <li><a href="#">npm Enterprise</a></li>
+                <li><a href="#">Products</a></li>
+                <li><a href="#">Solutions</a></li>
+                <li><a href="#">Resources</a></li>
+                <li><a href="#">Docs</a></li>
+                <li><a href="#">Support</a></li>
+              </ul>
+            </nav>
+        </div>
+
+        <div>  
+                <img class="logo" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\npm-logo.png" alt="logo"/>  
+              <form class="search-button" action="action_page.php">
+                <img class="search-bar" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\magnifying-glass.png" alt="search bar pic"/>
+                <input type="text" placeholder="Search packages" name="search">
+                <button type="submit" class="search-box">
+                    <i class="fa fa-search">Search</i>
+                 </button>
+              </form>
+                 <div>
+                    <a class="link-btn" id="join-button" href="#">Join</a>
+                    <a class="link-btn" href="#">Log In</a>
+                 </div>
+              
+        </div> 
+      </header>
+      <section id="cream-background">
+        <h1>Build amazing things</h1>
+        <article>
+          <p>Essential JavaScript development tools that help you <br> 
+            go to market faster and build powerful applications </br> using modern open source code.</p>
+        </article>
+        <div>
+          <a class="button" id="see-btn" href="#">See plans</a>
+          <a class="button" id="join-btn" href="#">Join for free</a>
+        </div>
+      </section>
+      <section id="info">
+        <img id="hexagon" src="C:\Users\joask\Desktop\prework-lab\lab-html-exercise\images\triangle-hexagon.svg" alt="triangle hexagon"/>
+        <h3>Bring the best of open source to your company</h3>
+        <article>
+          <p>npm is the tool used by over 11,000,000 JavaScript developers around the world. Your developers <br> 
+            already use it. Your company depends on it. Create an Org and get more out of the tools your <br>
+            team already knows and loves.</br></p>
```

Comment:
Instead of using <br> tags here to move text to the next line, you should set up the styling to achieve the same effect. That way it would be responsive.